opentech/apply/funds/workflow.py

```diff
@@ -79,60 +113,48 @@ def __str__(self):
         return self.name
 
 
-class BasePermissions:
-    def can_edit(self, user: 'User') -> bool:
-        if user.is_apply_staff:
-            return self.can_staff_edit(user)
-
-        if user.is_applicant:
-            return self.can_applicant_edit(user)
+class Permissions:
```

Comment:
I like the changes I have made to permissions generally here. they are easier to compose together.

However I think the addition of the can_view permission is not helpful. Probably adding a `publicly_visible` property to the phase might be more reasonable.

`hidden_from_applicant_permissions` just feels so clunky. especially as it inherits the default and overrides it.

Reply:
What don't you like about `hidden_from_applicant_permissions`?